test/commands/org/login/access-token.test.ts

```diff
@@ -28,99 +30,103 @@ describe('org:login:access-token', () => {
     custom_domain: 'https://foo.bar.org.salesforce.com',
   };
   /* eslint-enable camelcase */
+  let stubSfCommandUxStubs: ReturnType<typeof stubSfCommandUx>;
+  let prompterStubs: ReturnType<typeof stubPrompter>;
 
-  async function createNewStoreCommand(
-    flags: string[] = [],
-    promptAnswer = accessToken,
-    authFileExists = false,
-    useSfdxAccessTokenEnvVar = false
-  ): Promise<Store> {
-    authFields = {
-      accessToken,
-      instanceUrl: 'https://foo.bar.org.salesforce.com',
-      loginUrl: 'https://foo.bar.org.salesforce.com',
-      orgId: '00D000000000000',
-      username: 'foo@baz.org',
-    };
-
-    stubMethod($$.SANDBOX, StateAggregator, 'getInstance').resolves({
-      orgs: {
-        exists: () => Promise.resolve(authFileExists),
-      },
-    });
-    stubMethod($$.SANDBOX, Store.prototype, 'saveAuthInfo').resolves(userInfo);
-    stubMethod($$.SANDBOX, AuthInfo.prototype, 'getUsername').returns(authFields.username);
-    stubMethod($$.SANDBOX, AuthInfo.prototype, 'getFields').returns({
+  beforeEach(() => {
+    // @ts-expect-error because private method
+    $$.SANDBOX.stub(Store.prototype, 'saveAuthInfo').resolves(userInfo);
+    $$.SANDBOX.stub(AuthInfo.prototype, 'getUsername').returns(authFields.username);
+    $$.SANDBOX.stub(AuthInfo.prototype, 'getFields').returns({
       accessToken,
       orgId: authFields.orgId,
       instanceUrl: authFields.instanceUrl,
       loginUrl: authFields.loginUrl,
       username: authFields.username,
     });
-    stubMethod($$.SANDBOX, Store.prototype, 'getUserInfo').resolves(AuthInfo.prototype);
-    if (useSfdxAccessTokenEnvVar) {
-      stubMethod($$.SANDBOX, Env.prototype, 'getString').callsFake(() => accessToken);
-    }
-    const store = new Store(
-      [...new Set(['--instance-url', 'https://foo.bar.org.salesforce.com', '--no-prompt', ...flags])],
-      {} as Config
-    );
-    // @ts-ignore
-    $$.SANDBOX.stub(Store.prototype, 'askForAccessToken').resolves(promptAnswer);
-
-    return Promise.resolve(store);
-  }
+    // @ts-expect-error because private method
+    $$.SANDBOX.stub(Store.prototype, 'getUserInfo').resolves(AuthInfo.prototype);
+    stubSfCommandUxStubs = stubSfCommandUx($$.SANDBOX);
+    prompterStubs = stubPrompter($$.SANDBOX);
+  });
 
   it('should return auth fields after successful auth', async () => {
-    const store = await createNewStoreCommand(['--instance-url', 'https://foo.bar.org.salesforce.com', '--no-prompt']);
-    const result = await store.run();
-    expect(result).to.deep.equal(authFields);
-  });
+    prompterStubs.secret.resolves(accessToken);
 
-  it('should prompt for access token when accesstokenfile is not present', async () => {
```

Comment:
I don't know what `accesstokenfile` refers to.  It's not in help, docs.  Hallucination?